samples/conan/basic_example/main.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <iostream>
+#include <openvino/openvino.hpp>
+
+int main(int argc, char *argv[])
+{
+    try
+    {
```

Comment:
_applies to the entire file_
Does style checks accept that? Shouldn't it be:
```
int main(int argc, char *argv[]) {
    try {
```

Reply:
Formatting changed